.gitlab-ci.yml

```diff
@@ -12,11 +12,6 @@ variables:
     description: "Set to true to override rules in the reliability-env pipeline (e.g. override 'only deploy master')"
     
 build:
-  only:
```

Comment:
Make sure to revert this before merging 😉

Reply:
yep :)